devices/org.osbuild.loopback

```diff
@@ -71,10 +77,9 @@ class LoopbackService(devices.DeviceService):
         lock = UdevInhibitor.for_device(lo.LOOP_MAJOR, lo.minor)
         lo.on_close = lambda _l: lock.release()
 
-    def make_loop(self, fd: int, offset, sizelimit, lock):
+    def make_loop(self, fd: int, offset, sizelimit, lock, partscan):
         if not sizelimit:
-            stat = os.fstat(fd)
-            sizelimit = stat.st_size - offset
+            sizelimit = os.fstat(fd).st_size - offset
```

Comment:
yeah it's kind of confusing, basically because we now `import stat` using `stat` as the name of the variable here gives a warning. I realized we didn't really need a variable here so I just eliminated the use of it to also eliminate the problem :)

Reply:
Aha, nice one! Thanks for the explaination